mk/config.mk

```diff
@@ -275,6 +275,16 @@ CFG_DT ?= n
 # editing of the supplied DTB.
 CFG_DTB_MAX_SIZE ?= 0x10000
 
+# Device Tree Overlay support.
+# This define enables support for an OPTEE provided DTB overlay.
+# One of two modes is supported in this case:
+# 1. Append OPTEE nodes to an existing DTB overlay located at CFG_DT_ADDR or
```

Comment:
s/OPTEE/OP-TEE/g   (2 occurrences)

Sorry for such details. config.mk already contain OP-TEE references and your change is better if consistent with the whole file.